qldbshell/qldb_shell.py

```diff
@@ -47,27 +45,30 @@ class QldbShell:
 
     :type profile: str
 
-    :type pooled_driver: PooledQldbDriver
+    :type driver: QldbDriver
 
     """
 
-    def __init__(self, profile="default", pooled_driver=None):
+    def __init__(self, profile="default", driver=None, show_stats=False):
```

Comment:
Hmm this sets show_stats=false but the argument parser will pass in true?

Reply:
It's the default parameter. Not 100% necessary but there nonetheless.